src/fastpath/fastpath.c

```diff
@@ -465,7 +474,7 @@ void NORETURN fastpath_reply_recv(word_t cptr, word_t msgInfo)
     } else {
 #ifdef CONFIG_KERNEL_MCS
         /* Update queue. */
-        tcb_queue_t queue = tcbEPAppend(NODE_STATE(ksCurThread), ep_ptr_get_queue(ep_ptr));
+        tcb_queue_t queue = tcbIPCAppend(NODE_STATE(ksCurThread), ep_ptr_get_queue(ep_ptr));
```

Comment:
I prefer tcbAppend, because "IPC" is always ambiguous whether that includes notifications or not. E.g. in the manual it are separate chapters.